cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=100)
+    description = serializers.CharField(required=True)
+    duration = serializers.IntegerField(required=True)
```

Comment:
Consider using `PositiveIntegerField` for the `duration` field to ensure only positive values are accepted, as a movie duration cannot be negative.